### PR TITLE
Updated docs to reflect changes in the API

### LINKED ISSUE #500: Documentation sanity check
After substantial changes to our API we should see that `endpoints` , sample `json` , descriptions, etc. are in sync with what we are currently serving: 

Issues causing API changes:
- [x] https://github.com/gnosis/safe-client-gateway/issues/470 (`ETHER` to `NATIVE_COIN`)
- [x] https://github.com/gnosis/safe-client-gateway/issues/484 (`AddressEx` used everywhere)
- [x] https://github.com/gnosis/safe-client-gateway/issues/487 (Introduction of polymorphism to `ExecutionInfo` for `TxSummary`)
- [x] https://github.com/gnosis/safe-client-gateway/issues/490 (change any occurrence of `url` to `uri`)
- [x] https://github.com/gnosis/safe-client-gateway/issues/498 (changed query parameter `page_url` to `cursor`)

### PR DESCRIPTION
**Pull request by jpalvarezl**
Closes #500 

Modified examples to highlight changes in the API

### CODE REVIEW

File: src/models/service/transactions/details.rs [hunk 1]
```diff
@@ -73,21 +73,25 @@ use std::collections::HashMap;
...
+///     },
+///     "recipient": {
+///       "value": "0x126ab4d9e87b5cba98Ddeb75Df703E83500b6B7f"
+///     },
 ///     "direction": "INCOMING",
 ///     "transferInfo": {
-///       "type": "ETHER",
-///       "value": "50000000000000"
+///       "type": "NATIVE_COIN",
+///      "value": "100000000000000000"
```
Comment (fmrsabino) [inline src/models/service/transactions/details.rs:89]: nitpick: space was removed here

File: src/models/service/transactions/summary.rs [hunk 2]
```diff
@@ -16,21 +16,27 @@ use serde::Serialize;
...
 ///         "id": "multisig_0x126ab4d9e87b5cba98Ddeb75Df703E83500b6B7f_0x28b4cc29c036c2df40a1ba8d684cdab736abaf7d5cb84b217428462a2b4e3318",
 ///         "timestamp": 1604700419000,
 ///         "txStatus": "SUCCESS",
 ///         "txInfo": {
 ///           "type": "Custom",
-///           "to": "0x8D29bE29923b68abfDD21e541b9374737B49cdAD",
+///           "to": {
+///             "value": "0x8D29bE29923b68abfDD21e541b9374737B49cdAD",
+///             "name": "Gnosis Safe: Multi Send 1.1.1",
+///             "logoUri": "https://safe-transaction-assets.staging.gnosisdev.com/contracts/logos/0x8D29bE29923b68abfDD21e541b9374737B49cdAD.png"
```
Comment (fmrsabino) [inline src/models/service/transactions/summary.rs:30]: Do we have a production URI for this one?

File: src/models/service/about.rs [hunk 3]
```diff
@@ -9,7 +9,7 @@ use serde::Serialize;
 /// {
 ///   "transactionServiceBaseUri": "https://safe-transaction.staging.gnosisdev.com/api/v1",
```
Comment (fmrsabino) [inline src/models/service/about.rs:10]: Maybe we should use the production URL for documentation?

File: src/models/service/transactions/summary.rs (hunk 2 above)
Comment (jpalvarezl) [reply src/models/service/transactions/summary.rs:30]: I am not sure, as I just pasted the json I took from the service itself. I think the value being from either staging or prod doesn't matter in the scope of documentation.

File: src/models/service/about.rs (hunk 3 above)
Comment (jpalvarezl) [reply src/models/service/about.rs:10]: Same as above, environment shouldn't matter for the sake of documentation, but I would add the `/chains/4/` or something like that to reflect what could actually be returned.

**Review by fmrsabino — APPROVED** [review]

**Review by fmrsabino — APPROVED** [review]

**Pull request merged**